### PR TITLE
Release Version 1.8

### PR DESCRIPTION
**Pull request by t-oster**
This Pull-Request shall be the place for discussion for the 1.8 release. I think it is time to do this release, because there are many changes since 1.7.

Please comment which Bugs/Features you think have to be resolved before this release should be done.

I will update this description with the things to do:
## TODO:
- [ ] select and fix important bugs:
  - [ ] #174 Scaling 1.25 off
  - [x] Bugs specific to Windows
    - [x] #165 Windows installer does not detect JRE on Windows 7/8
    - [x] #194 Windows installer crash my path
  - [ ] Bugs specific to OSX
    - [ ] #171 Severe UI malfunction und Mac OS X 10.7.5
- [ ] update changelog

### CODE REVIEW

**Comment by renebohne** [comment]
Can we make sure that SVG import scaling is free of bugs before we release 1.8?
I think that issue #174 is a very important thing, because people in our fab lab not only create SVGs with Illustrator or Inkscape, but with any software. We have many problems with scaling and need to be very careful before we actually start cutting.

**Comment by renebohne** [comment]
These features that have a deadline of 4.2.2015:
thingiverse integration #221 
automatic arrangement of shapes #222 
facebook integration #223 
SVG Exporter #225 
TSP Optimizer #226 
Usability Improvements #227 

Since they are developed by students, we need to have some time for testing... but if the release can wait until 1.3.15, I would love to see these features in the major version 1.8 :)

**Comment by mgmax** [comment]
My opinion is that we should skip the upcoming features, because they will inevitably have bugs that do not appear immediately.
The release should be as stable as possible - if users are interested in new features, they can just use the develop version.

**Comment by renebohne** [comment]
ok... so what are we waiting for? What has to be done for 1.8?
To me, only issue #174 is a serious thing that is nothing I would expect in a stable release. 
What are the other things?
#171 : I can't reproduce this bug anymore. Mac OS X 10.9.5. Maybe someone with 10.10 can test if there are problems with it? Otherwise, I think that this bug got fixed...

I can't say anything about the two windows bugs right now, but I will install windows this weekend and give it a try.

**Comment by mgmax** [comment]
@t-oster please update the description. #165 is fixed, the core problem of #194 is also fixed.

File: README.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ www.visicut.org
 
 Download
 --------
-* Latest stable release: http://137.226.142.25:5080/wwwshare/VisiCutNightly/master
```
Comment (brains12345) [inline README.md]: hoping etching problem solved

**Comment by mgmax** [comment]
As it's been ages since the last release, I would suggest switching to a "rolling release" scheme:
In a certain time interval, the development branch is frozen and declared as release candidate. A pull request then allows for discussion and feedback by beta testers. If after maybe three weeks there is positive feedback and no regressions were discovered, we now have a new release.

After some time, this can be repeated for the next release. 

As a contribution, I would offer beta-testing these versions at FAU FabLab, as we are currently using random snapshots of the development branch.

**Comment by mgmax** [comment]
It's been ages since the last release, and all recent changes are bugfixes. I bet the current development version is better than the previous release in (almost?) all aspects and that the two 'release blocker' bugs #171 and #174 were already present before.

I think it's time to release; or just automatically set the version number to (year-2000)/10, which would mean releasing 1.8 in 2018 :-)

**Comment by t-oster** [comment]
I think the separation of "develop" aka "unstable" and master aka release does not make sense anymore. I am merging develop to master and bump the version to 1.8. If we have really experimental features, they can be compiled in a separate branch as testing version.

**Pull request merged**